README.md

```diff
@@ -48,9 +48,11 @@ Install uv with our standalone installers, or from [PyPI](https://pypi.org/proje
 ```shell
 # On macOS and Linux.
 curl -LsSf https://astral.sh/uv/install.sh | sh
+curl -LsSf https://astral.sh/uv/0.2.8/install.sh | sh
```

Comment:
Slightly worried this is ambiguous and people will run both

Reply:
So perhaps https://github.com/astral-sh/uv/pull/4105/commits/d3275dbfbf6ad9194ef407628bccc685006e01ad